src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -117,6 +117,46 @@ CustomErrorComponent.getInitialProps = async contextData => {
 export default CustomErrorComponent;
 ```
 
+```typescript {filename:pages/_error.ts}
+/**
+ * NOTE: This requires `@sentry/nextjs` version 7.3.0 or higher.
+ *
+ * This page is loaded by Nextjs:
+ *  - on the server, when data-fetching methods throw or reject
+ *  - on the client, when `getInitialProps` throws or rejects
+ *  - on the client, when a React lifecycle method throws or rejects, and it's
+ *    caught by the built-in Nextjs error boundary
+ *
+ * See:
+ *  - https://nextjs.org/docs/basic-features/data-fetching/overview
+ *  - https://nextjs.org/docs/api-reference/data-fetching/get-initial-props
+ *  - https://reactjs.org/docs/error-boundaries.html
+ */
+
+import * as Sentry from "@sentry/nextjs";
+import { NextPage } from "next";
+import NextErrorComponent, { ErrorProps } from "next/error";
```

Comment:
Optional nit: I'm slowly over time getting more and more on the `import type` bandwangon. 

```suggestion
import type { NextPage } from "next";
import type { ErrorProps } from "next/error";
import NextErrorComponent from "next/error";
```